### PR TITLE
Adopt spec to #248

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

**Comment by hennevogel** [comment]
@ChrisBr just some new quotes in the stubbed requests...

File: spec/influxdb/cases/query_cluster_spec.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@
   describe "#create_cluster_admin" do
     let(:user) { 'adminadmin' }
     let(:pass) { 'passpass' }
-    let(:query) { "CREATE USER #{user} WITH PASSWORD '#{pass}' WITH ALL PRIVILEGES" }
+    let(:query) { "CREATE USER \"#{user}\" WITH PASSWORD '#{pass}' WITH ALL PRIVILEGES" }
```
Comment (ChrisBr) [inline spec/influxdb/cases/query_cluster_spec.rb:20]: Why is pass in single quotes and user in double quotes?

**Review by ChrisBr — COMMENTED** [review]
[nitpick] Should we swap to single quotes so we don't need to escape the double quotes always?

**Review by ChrisBr — APPROVED** [review]

File: spec/influxdb/cases/query_cluster_spec.rb (hunk 1 above)
Comment (hennevogel) [reply spec/influxdb/cases/query_cluster_spec.rb:20]: Because that's what we generate now. See #248

**Comment by hennevogel** [comment]
> [nitpick] Should we swap to single quotes so we don't need to escape the double quotes always?

Yeah let's unbreak master first though...

**Pull request merged**